### ISSUE TITLE
createClass deprecation from React 15.5

### ISSUE THREAD

**Issue by jpsear** [twitter-fabric/velocity-react#184]
Just a heads up

```
warning.js:36 Warning: VelocityComponent: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.
```

https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.createclass

**Comment by finneganh** [comment]
Thanks for the heads-up! I'm planning on fixing this fairly soon.

**Comment by finneganh** [comment]
#191 has fixes for this if you're able to try it out. I had to change some of the internals and also add a prepublish step to this library in order to use ES2015 classes so i want to do a careful rollout.

**Comment by finneganh** [comment]
Fixed in v1.3.0. Thanks for your patience!
